.travis.yml

```diff
@@ -304,7 +307,9 @@ before_install:
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install ccache bison flex protobuf@3.1 liblo libmicrohttpd; fi # ossp-uuid, homebrew/python/numpy and libusb already present
```

Comment:
The protobuf entry wants dropping from here as it's covered below now.

Reply:
True, fixed